models/yang/extensions/openconfig-sampling-sflow-deviation.yang

```diff
@@ -0,0 +1,54 @@
+module openconfig-sampling-sflow-deviation {
```

Comment:
ok it will be taken care in next commit, just verified deviation is copied to target directory
root@sonic:/usr/models/yang# ls *sflow*
openconfig-sampling-sflow-annot.yang  openconfig-sampling-sflow.yang  sonic-sflow.yang
root@sonic:/usr/models/yang# exit

after change

root@sonic:/usr/models/yang# ls *sflow*
openconfig-sampling-sflow-annot.yang  openconfig-sampling-sflow-deviation.yang  openconfig-sampling-sflow.yang  sonic-sflow.yang
root@sonic:/usr/models/yang#

Reply:
This will create conflict as it depends on https://github.com/sonic-net/sonic-mgmt-common/pull/89/files#diff-d832e5808954c6cf44be03eaa0e64428aecb80ad00129008a561c9c995e4e0fd merge first.

Reply:
That should not be a problem. For this PR tests to succeed, both #89 and #87 should be committed first and then merged to this branch. Conflict can be resolved at that time.